src/main/java/com/cloudbees/jenkins/support/SupportAction.java

```diff
@@ -517,7 +520,7 @@ public ProgressiveRendering getGeneratorByTaskId(String taskId) throws Exception
     }
 
     public static class SupportBundleAsyncGenerator extends ProgressiveRendering {
-        private final Logger logger = Logger.getLogger(SupportBundleAsyncGenerator.class.getName());
+        private final Logger logger = Logger.getLogger(SupportAction.class.getName());
```

Comment:
No I meant you could just use https://github.com/jenkinsci/support-core-plugin/blob/effed5e2e7f6fcf6edeb6e562795bd3524c4c9ed/src/main/java/com/cloudbees/jenkins/support/SupportAction.java#L105